.github/auto-label.yml

```diff
@@ -260,6 +260,9 @@ vendor/kubectl-1.24:
 vendor/kubectl-1.25:
 - any: ["vendor/kubectl-1.25/**"]
   all: ["!bin/**", "!tasks/**"]
+vendor/kubectl-1.26:
+- any: ["vendor/kubectl-1.26/**"]
+  all: ["!bin/**", "!tasks/**"]
```

Comment:
wouldn't you want to remove EOL kubectl above?

Reply:
This file is automatically generated, and there is no need to remove the configuration for a package. If for some reason something changes, we still want to label the PR appropriately.